src/js/components/Overview/MakeChartCard.js

```diff
@@ -50,7 +51,13 @@ const MakeChartCard = ({ section, chart, onRemoveChart }) => {
   return (
     <div key={name} style={{ height: '100%', width: '430px' }}>
       <Card title={t(title)} style={CARD_STYLE} size="small" extra={<Space size="small">{ed}</Space>}>
-        <Chart chartType={chartType} data={data} units={config?.units || undefined} />
+        {data.filter((e) => !(e.x === 'missing')).length !== 0 ? (
+          <Chart chartType={chartType} data={data} units={config?.units || undefined} />
+        ) : (
+          <Row style={{ height: '350px ' }} justify="center" align="middle">
+            <CustomEmpty text="No Data" />
```

Comment:
need to add translation term for `No Data`

Reply:
oh wait maybe no results

Reply:
done